<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Christian Schenk</author>
 <class>SiteWizType</class>
 <widget class="QWizardPage" name="SiteWizType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="title">
   <string>Installation Source</string>
  </property>
  <property name="subTitle">
   <string>Choose where you want to install packages from.</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QRadioButton" name="rbRemote">
       <property name="text">
        <string>Packages shall be installed from the &amp;Internet.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnectionSettings">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Connection Settings...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chkMiKTeXNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Get pre-release (experimental) packages.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rbLocal">
     <property name="text">
      <string>Packages shall be installed from a &amp;directory.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbCD">
     <property name="text">
      <string>Packages shall be installed from a &amp;MiKTeX CD/DVD.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbLocal</tabstop>
  <tabstop>rbCD</tabstop>
  <tabstop>btnConnectionSettings</tabstop>
  <tabstop>chkMiKTeXNext</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbRemote</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnConnectionSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRemote</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkMiKTeXNext</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
